packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -138,18 +138,24 @@ export const styles = css`
         flex-direction: column;
         overflow-y: auto;
         width: 100%;
+        --ni-private-listbox-num-visible-options: 10.5;
+        --ni-private-listbox-anchor-element-gap: ${smallPadding};
         --ni-private-listbox-padding: ${smallPadding};
         --ni-private-listbox-filter-height: 0px;
         --ni-private-listbox-loading-indicator-height: 0px;
         max-height: min(
             calc(
-                ${smallPadding} + 2 * ${borderWidth} + ${controlHeight} * 10.5 +
+                var(--ni-private-listbox-anchor-element-gap) + 2 *
```

Comment:
I find that the wrapping here makes visually parsing & understanding the logic really difficult. I assume prettier/eslint is fighting you here? Would it be reasonable to disable those for these calculations so that you could write it in a way where the different pieces of the calculation are more apparent. I'm thinking something like this:
```
gap +
2 * borderWidth +
padding +
controlHeight * numVisibleOptions +
filterHeight +
loadingIndicatorHeight
```